compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1962,6 +1962,9 @@ error.worker.send.action.not.allowed.in.lock.statement=\
 error.worker.receive.action.not.allowed.in.lock.statement=\
   using receive action within the lock statement is not allowed to prevent possible deadlocks
 
+error.named.arg.not.allowed.for.rest.param=\
+  named argument is not allowed for rest parameter
```

We once had chat regarding this and IIRC conclusion was to make the error message shorter.
Other programming languages seem to follow the same.

```suggestion
  named argument not allowed for rest parameter
```

---

Fixed.